src/backend/aspen/workflows/nextstrain_run/builder_base.py

```diff
@@ -37,9 +37,11 @@ def update_build(self, config):
             self.tree_build_level = "country"
         elif not location.location:
             self.tree_build_level = "division"
-        # Fill out region/country/division/location fields if the group has them,
+        # The region field for tree build needs to be 'global' to avoid country names being converted to region
+        build["region"] = "global"
```

Comment:
Can we put this change in the YAML template instead if it's going to be the same for all tree runs?

Reply:
I left it here thinking there is a slim chance this will become a parameter at some point (but not in a foreseeable future). Should I put it into the template and move it back when we actually use is as a parameter?

Reply:
Yeah this makes more sense. Let me move it~~~